cluster-autoscaler/core/scale_down.go

```diff
@@ -538,6 +539,7 @@ func getEmptyNodes(candidates []*apiv1.Node, pods []*apiv1.Pod, maxEmptyBulkDele
 		var available int
 		var found bool
 		if available, found = availabilityMap[nodeGroup.Id()]; !found {
+			// Will be cached.
```

Comment:
What do you mean?

Reply:
The result of TargetSize will be cached and we will not invoke it for the same nodegroup.